python/Homework.py

```diff
@@ -0,0 +1,210 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn import tree
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
```

Comment:
Please use `def main()` here and `if __name__ == "__main__"...` at the end of your code.

Reply:
I updated! Thank you for your comments!